akka-actor/src/main/scala/akka/util/Helpers.scala

```diff
@@ -14,7 +14,9 @@ import java.util.Locale
 
 object Helpers {
 
-  val isWindows: Boolean = System.getProperty("os.name", "").toLowerCase(Locale.ROOT).indexOf("win") >= 0
+  def toLowerCase(s: String) = s.toLowerCase(Locale.ROOT)
```

Comment:
but then I'd like it to be used consistently in all places the previous style was used.
Not quite sold on this method though - can we in this PR only do the findClassLoader move please?